### DISCUSSION TITLE
what version of willow to build to work with WIS ng branch?

### DISCUSSION THREAD

**Discussion by hamishcunningham** [toverainc/willow#198] [General]
Hello! I'm running the wisng branch and the latest main branch of willow, but think configuration of WIS URLs may have changed? At any rate I get 
```
I (15:43:10.021) WILLOW: Using WIS URL 'https://localhost:19000/api/willow'
I (15:43:10.023) WILLOW: WIS HTTP client starting stream, waiting for end of speech
E (15:43:10.033) esp-tls: [sock=54] delayed connect error: Connection reset by peer
E (15:43:10.038) esp-tls: Failed to open new connection
E (15:43:10.043) TRANSPORT_BASE: Failed to open a new connection
E (15:43:10.051) HTTP_CLIENT: Connection failed, sock < 0
E (15:43:10.056) AUDIO_ELEMENT: [http_stream_writer] AEL_STATUS_ERROR_OPEN,28674
```
on willow, and wondering if I need to use a different branch or different config or...?

Tnx!

**Comment by nikito** [comment]
Think the problem is the url. Looks like it is set to localhost, but needs to be set to the hostname of your wis server; localhost on the willow Device would tell it to connect to itself. 🙂

**Comment by hamishcunningham** [comment]
Duh!
Thanks @nikito !
